1-js/02-first-steps/05-types/article.md

```diff
@@ -82,8 +82,9 @@ const bigInt = 1234567890123456789012345678901234567890n;
 As `BigInt` numbers are rarely needed, we don't cover them here, but devoted them a separate chapter <info:bigint>. Read it when you need such big numbers.
 
 ```smart header="Compatibility issues"
-Right now `BigInt` is supported in Firefox/Chrome/Edge, but not in Safari/IE.
+Right now `BigInt` is supported in the latest version Firefox/Chrome/Edge and Safari with iOS 14, but not in IE.
```

Comment:
I think we don't need to mention Safari's version, as we don't mention it for others too. Instead, they can look it up in caniuse or MDN. What do you think?

```suggestion
Right now `BigInt` is supported in Firefox/Chrome/Edge/Safari, but not in IE.
```